jarbas/core/management/commands/companies.py

```diff
@@ -45,6 +48,12 @@ def save_companies(self):
                 self.count += 1
                 self.print_count(Company, count=self.count)
 
+    @staticmethod
+    def transform(row):
+        return row._asdict()
```

Comment:
I don't think we need a method for that… calling `_asdict()` in the `for` loop seams enough.